content/swiftui/concepts/viewmodifier/terms/underline/underline.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '.underline()'
+Description: 'A ViewModifier that applies an underline style to text characters in a View.'
+Subjects:
+- 'Mobile Development'
+- 'iOS'
+Tags:
+- 'SwiftUI'
+- 'SwiftUI Modifiers'
+CatalogContent:
+- 'learn-swift'
+- 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.underline()`** view modifier in SwiftUI applies an underline style to text characters in a View.
+
+## Syntax
+
+```psuedo
+Text("This text will be underlined")
+.underline()
+```
+
+The `.underline()` modifier is applied to all characters within the `Text` (which conforms to View protocol).
+
+## Example
+
+The following example creates some underlined text:
+
+```swift
+var body: some View {
+Text("Underlined text")
+.underline()
+}
+```
```

Comment:
Need one line break after the code block.

Reply:
Code block highlighted with a line break.